bin/integration-cleanup

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+
+# Age tolerance threshold for running instances (in seconds); defaults to 4 hours.
+#   Override in CircleCI by defining INSTANCE_AGE_THRESHOLD for .env.integration.
+AGE_THRESHOLD=${INSTANCE_AGE_THRESHOLD:-"14400"}
+
+export OS_USERNAME=$OPENSTACK_USER
+export OS_PASSWORD=$OPENSTACK_PASSWORD
+export OS_TENANT_NAME=$OPENSTACK_TENANT
+export OS_AUTH_URL=$OPENSTACK_AUTH_URL
+
+echo "Querying Nova for list of running integration test instances..."
+
+# Get the list of potential instances to terminate.
+# Returns a table of instances with four columns:
+#   The instance ID, key pair, creation date, and name.
+INSTANCE_LIST=$(nova list --fields=key_name,created,name --status=ACTIVE | grep circleci | sed 's/|//g')
+
+if [ ${#INSTANCE_LIST} -eq 0 ]
+then
+    echo "No active instances returned, nothing to clean up. Quitting."
+    exit
+fi
+
+INSTANCE_LIST_SIZE=$(echo "$INSTANCE_LIST" | wc -l)
```

Comment:
@bdero 

> The issue here is that `wc -l` simply counts newline characters, and if nova returns nothing or we grep to nothing, the var is still stored with one newline character. That's why I have a check that just exits if the string length of the nova results is 0.

OK, that makes sense. I'm inclined to suggest adding a comment with this explanation above this line (L28), but then again this might be totally obvious to someone with more robust bash scripting skills than me :) I'll leave the decision up to you.